RadarSDK/RadarUtils.h

```diff
@@ -21,13 +21,15 @@ NS_ASSUME_NONNULL_BEGIN
 + (NSString *)deviceType;
 + (NSString *)deviceMake;
 + (BOOL)locationBackgroundMode;
++ (NSString *)locationAuthorization;
++ (NSString *)locationAccuracyAuthorization;
```

Comment:
This is nullable I think

Reply:
Good catch, will specify

Reply:
Changed this to return `FULL` pre-iOS 14, so never `nil`